pom.xml

```diff
@@ -240,6 +242,20 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
```

Comment:
Shouldn't this go in a "profile" section like we do here - https://github.com/HotelsDotCom/waggle-dance/blob/master/pom.xml#L209 since we only want to do the signing for a release? @ddcprg?

Reply:
yes - my bad